qiskit/transpiler/passes/synthesis/plugin.py

```diff
@@ -245,6 +245,7 @@ def run(self, high_level_object, **options):
    UnitarySynthesisPlugin
    UnitarySynthesisPluginManager
    unitary_synthesis_plugin_names
+   SolovayKitaevSynthesis
```

Comment:
You can't ask Sphinx to document this class in more than one place.  It also wouldn't succeed documented like this from here at the moment anyway, because it's not imported into this module.